### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by OsypenkoVlad**
Add Task 3!

### CODE REVIEW

File: Calculator/src/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String operator;
+        double result;
+
+        System.out.println("First value:");
+        double firstvalue = CorrectNumb();
```
Comment (Hmeliumm) [inline Calculator/src/Calculator.java]: please move variable initialization in one place (like where result)

File: Calculator/src/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+        System.out.println("Console operation:");
+        operator = sc.next();
+
+        System.out.println("Second value:");
+        double secondvalue = CorrectNumb();
```
Comment (Hmeliumm) [inline Calculator/src/Calculator.java]: please move variable initialization in one place (like where result)

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Calculator/src/Calculator.java (hunk 1 above)
Comment (OsypenkoVlad) [reply Calculator/src/Calculator.java]: I have corrected

File: Calculator/src/Calculator.java (hunk 2 above)
Comment (OsypenkoVlad) [reply Calculator/src/Calculator.java]: I have corrected

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**